src/Features/CSharpTest/Wrapping/InitializerExpressionWrappingTests.cs

```diff
@@ -192,6 +225,16 @@ class C {
                 }
                 """,
                 """
+                class C {
+                    void Bar() {
+                        var test = new[]
+                        {
+                            "the", "quick", "brown", "fox", "jumps", "over", "the", "lazy", "dog", "the", "quick", "brown", "fox", "jumps", "over", "the", "lazy", "dog"
+                        };
```

Comment:
improvement in overall feature.  we were not offering this option before in the case where the lower option was offered.